dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -60,6 +60,7 @@
     "kafka",
     "qdrant",
     "mssql",
+    "drill",
```

Comment:
NIT: can we sort these alphabetically :D  ? It hurts a bit looking at the list being semi-random order.

Reply:
Fixed, please don't take a look at the rest of the file :)

Reply:
My approach is to fix and sort what I touch :). If you look at some of my changes from last months many of them actually did some sorting. And the thing is ....

Let's make every change leave our small "world" in a slightly better shape than what we got before - **soriting out** one thing at a time :D.